src/basic/time-util.c

```diff
@@ -459,6 +459,24 @@ int dual_timestamp_deserialize(const char *value, dual_timestamp *t) {
         return 0;
 }
 
+int deserialize_timestamp_value(const char *value, usec_t *timestamp) {
+        unsigned long long a;
+
+        assert(value);
+
+        if (!value)
+                return 0;
+
+        if (sscanf(value, "%llu", &a) != 1) {
+                log_debug("Failed to parse finish timestamp value %s.", value);
+                return -EINVAL;
+        }
+
+        *timestamp = a;
+
+        return 0;
```

Comment:
This case is not distinguishable from the bail above. It doesn't really matter because you don't check the return values of this function anyway, but the usual coding style in systemd for such occasions is to return `1` if something happened, and `0` otherwise.